src/tools/quemap/map.c

```diff
@@ -869,6 +889,8 @@ void LoadMapFile(const char *file_name) {
 	memset(map_planes, 0, sizeof(map_plane_t) * MAX_BSP_PLANES);
 	num_map_planes = 0;
 
+	memset(plane_hash, 0, sizeof(plane_hash));
```

Comment:
In a way it kind of sucks that we can't also have explicit .mat files for single textures like Q3 does, it'd make the top bit cleaner if it could just load them from, say, common/origin.mat for instance. Oh well.